developer/src/kmc-keyboard/src/compiler/messages.ts

```diff
@@ -54,7 +54,7 @@ export class CompilerMessages {
   static HINT_VkeyMapIsRedundant = SevHint | 0x000A;
 
   static Error_VkeyMapIsRepeated = (o:{vkey: string}) =>
-    m(this.ERROR_VkeyMapIsRepeated, `Virtual key '${o.vkey}' has more than one VkeyMap entry.`);
+    m(this.ERROR_VkeyMapIsRepeated, `Virtual key '${o.vkey}' has more than one vkey entry.`);
```

Comment:
this change of course isn't related to disp :grin:

Reply:
Will split